wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
-### Use of IP Addresses
+All communications between and among Clients, Servers, Directories, and third-party protocol users are sent without encryption.
 
-When you connect to Server, the Server operator can see your IP address while you are connected.  If the Server operator has enabled logging (which is off by default) your IP address will also be logged and stored in the Server's log file.
+## Jamulus Servers
 
-The IP addresses of all Servers registered with the Directory can also be seen by third parties for informational or other purposes (for example [here](https://explorer.jamulus.io/)). Your public IP address is otherwise not logged or stored by Jamulus, but may be stored or processed by third parties.
+When you connect to a Server, either directly or through a Directory, the operator of that Server is responsible for its operation policy, privacy policy, and data use policy. While unmodified Servers do not log or store your connection or profile information, some modified Servers may do this; therefore, you should have no expectation of privacy with respect to your profile information or internet address.
 
-### Audio Recordings
+## Chat Exchanges
 
-You will see a notice if you are connected to a Jamulus Server when Server recording is turned on. Recordings of each player are stored by the Server separately as .WAV files and only the Server operator has access to them unless they choose to make them available to third parties.
+Chats are textual messages that can be exchanged between Clients connected to the same Server. Everyone connected to a Server can see all chats and there should be no expectation of privacy with respect to information sent through the chat feature of Jamulus. While unmodified Servers do not log or store chats, some modified Servers may do this.
 
-### Text Chat
+## Audio Recordings
 
-When you type a message in the Chat Window, other connected players can see that, but chats are not stored by the Server and neither the Server operator nor any third parties have access to them.
+Unmodified Servers will display a notice if recording is turned on. Recordings of each player are stored by the Server and are controlled by the Server operator.
 
-### Data Transmission
+It is possible for connected Clients to make recordings of sessions outside of Jamulus itself, for which there may be no notice or indication. Jamulus has no way to detect or control these situations and makes no representations as to the collection or use of such recording data.
 
-Please note that all audio and text data is sent and received between the Jamulus Server and Client without encryption.
+## Directories
+
```

Comment:
... If you do not wish to send your information to the directories, start Jamulus from the command line to directly connect to a server.

(or something comparable)

Reply:
If I connect to a server using command line, doesn't that server report me to a directory if it's listed on one?

Reply:
Not sure what @ann0see means by this

Reply:
If someone does not like his IP to be published to the directory, they should not open the connect window at all. This they should start Jamulus with the -c flag to connect directly to the server they want to jam on.

Reply:
Oh OK, so we'd have to say something like "IP addresses of servers connected to public Directories are displayed on explorer.jamulus.io..." and then explain what to do? Privacy statements don't tend to also be instruction manuals but I guess we could?

Reply:
You're saying my IP _as a client_ is published to the directory unless I connect using -c ?

Reply:
> start the client with -c and never open the Connection Setup dialog. Right? The IP addresses of servers are published by the Directory. (Right?)

correct.

Reply:
Given that (as a non-technical person) you can only get the IP address of servers on Directories from Explorer, which ping times do not relate to your client, isn't the `-c` method pretty theoretical? How would you know which server to connect to?

Reply:
Yes, you'd need to look via Explorer or something else, unless you get the IP from someone. I'd assume that people who don't want to send their IP are using unregistered servers or know (mostly) what they're doing/get help.

Reply:
In that case it sounds like we need to say "If you do not wish to send your information to the Directories, you will need to connect to Servers directly" and leave them to read the manual I think. Otherwise any recommended usage methods in this doc start duplicating what's in the manual and it's all going to fall apart over time.

Reply:
Ok. Then write it like that.